website/learn/effective-elvish.md

```diff
@@ -121,14 +121,15 @@ newline, and the second value is a list. When we capture the output, we get
 those exact values back. Passing structured data is difficult with byte-based
 output, but trivial with value output.
 
-Besides `put`, many other builtin commands also write to structured output, like
-`splits`:
+Besides `put`, many other builtin commands and modules that ship with Elvish
```

Comment:
> ... modules ... write to structured output

modules -> commands in builtin modules

Reply:
Thanks, totally agree. I struggled with wording that!